apps/app/src/components/Dashboard/OrganizationDashboard/OrganizationCauses.tsx

```diff
@@ -1,7 +1,121 @@
-import React from 'react'
+import { FetchBalanceResult } from '@wagmi/core'
+import React, { useState } from 'react'
 
-const OrganizationCausesTab: React.FC = () => {
-  return <p>Organization Causes</p>
+import { GridItemTwelve, GridLayout } from '@/components/GridLayout'
+import { Card } from '@/components/UI/Card'
+import { Spinner } from '@/components/UI/Spinner'
+import { useAppPersistStore } from '@/store/app'
+
+import GetBalance from '../GetBalance'
+
+const OrganizationCauses: React.FC = () => {
+  const { currentUser } = useAppPersistStore()
+  const [vhrGoal, setVhrGoal] = useState(600) // use hardcoded goal for now
+
+  const [data, setData] = useState<FetchBalanceResult>()
+  const [isLoading, setIsLoading] = useState<boolean>(false)
+
+  const Progress = ({
+    progress,
+    total,
+    className
+  }: {
+    progress: number
+    total: number
+    className?: string
+  }) => (
+    <div className={className}>
+      <div className="w-full bg-gray-200 rounded-full h-5 ">
+        <div
+          className="bg-green-400 h-5 rounded-full"
+          style={{
+            width: `${Math.min(Math.trunc((progress / total) * 100), 100)}%`
+          }}
+        ></div>
+      </div>
+    </div>
+  )
+
+  return (
+    <GridLayout>
+      <GridItemTwelve>
+        <Card>
+          {currentUser && (
+            <GetBalance
+              address={currentUser.ownedBy}
+              callback={(data: FetchBalanceResult, isLoading: boolean) => {
+                setData(data)
+                setIsLoading(isLoading)
+              }}
+            />
+          )}
```

Comment:
If you check the latest version of main in the volunteer VHR tab, this chunk of code can be replaced by using a hook called useWalletBalance()

Reply:
Sorry but could you explain how to create a hook?